test/cli/testutils/strings.go

```diff
@@ -13,6 +13,11 @@ import (
 	manet "github.com/multiformats/go-multiaddr/net"
 )
 
+var (
+	AlphabetEasy = []rune("abcdefghijklmnopqrstuvwxyz01234567890-_")
+	AlphabetHard = []rune("abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ01234567890!@#$%^&*()-_+= ;.,<>'\"[]{}() ")
```

Comment:
Is this used anywhere?

Reply:
Yes, there is one cli test that uses it.